### PR TITLE
Cross Account sharing

### PR DESCRIPTION
**Pull request by srivipar**
Cross Account sharing

*Issue #, if available:*

*Description of changes:*
Cross Account sharing best practices

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/datasharing/cross-account/cafaqs.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+### 1. What are the various cross account topologies ?
+
+Lake formation cross account sharing feature is used for sharing catalog and underlying data lake access. Customer can share resource without duplicating data avoiding extensive pipelines to make the data available to other lines of business. These models enable various lines of business collaboration and require planning to standardize the enterprise level adaption.
+
+#### Hub and Spoke:
+
+Hub and Spoke model are chosen when customers want to centralizes metadata discovery and permission management in governance account across line of business. In this model domain accounts will share the data with central governance account that hosts the metadata for enterprises to discovery and request access for the resource without domain owners involved in the fulfilment process. Central governance account provides better audibility on resource access and scaling the permission across enterprise.  
+
+For more details on the setup refer to blog: 
+https://aws.amazon.com/blogs/big-data/introducing-aws-glue-crawlers-using-aws-lake-formation-permission-management/
```
Comment (hocanint-amzn) [inline content/datasharing/cross-account/cafaqs.md:12]: Not sure that this blog post is a good reflection of what a hub and spoke model looks like.

File: content/index.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # AWS Lake Formation
 
-AWS Lake Formation easily creates secure data lakes, making data available for wide-ranging analytics. This guide provides readers best practices when using various features and aspects of Lake Formation.
+AWS Lake Formation helps you centrally govern, secure, and globally share data for analytics and machine learning. This guide provides readers best practices when using various features and aspects of Lake Formation.
```
Comment (hocanint-amzn) [inline content/index.md:3]: Good catch!

**Review by hocanint-amzn — COMMENTED** [review]
Some comments mostly around hub and spoke definition.

File: mkdocs.yml [hunk 3]
```diff
@@ -14,6 +14,11 @@ nav:
       - Best Practices: 'lf-tags/best-practices.md'
       - Example Usecase: 'lf-tags/example-usecase.md'
       - Limitations: 'lf-tags/limitations.md'
+  - 2. Cross Account Data Sharing: 'datasharing/cross-account/index.md'
```
Comment (hocanint-amzn) [inline mkdocs.yml:17]: Please work with Aarthi to consolidate/reconcile the changes she made for data sharing. It seems like she made very similar changes.

Comment (srivipar) [reply mkdocs.yml]: ack

**Review by hocanint-amzn — APPROVED** [review]
Looks good. we will address any minor fixes before publishing the content.

**Pull request merged**